custom_components/automower/manifest.json

```diff
@@ -1,12 +1,12 @@
 {
   "domain": "automower",
   "name": "Automower",
-  "version": "0.5.1",
+  "version": "0.5.2",
   "config_flow": true,
-  "documentation": "https://github.com/astrandb/automower_hass",
-  "issue_tracker": "https://github.com/astrandb/automower_hass/issues",
+  "documentation": "https://github.com/Nordicfastware/automower_hass",
+  "issue_tracker": "https://github.com/Nordicfastware/automower_hass/issues",
```

Comment:
I don't see the point. The users should be pointed to this repo. Of course you are free to use and maintain you own fork if you prefer.